src/Lumi/Components/List.purs

```diff
@@ -11,18 +11,22 @@ import Lumi.Components.Size (Size(..))
 import React.Basic (Component, JSX, createComponent, element, makeStateless)
 import React.Basic.DOM as R
 
-type ListProps =
+type CommonListProps rest =
   { size :: Maybe Size
   , rightAligned :: Boolean
   , rows :: Array (Array JSX)
-  , borders :: Boolean
+  | rest
   }
 
+type ListProps = CommonListProps
+  ( borders :: Boolean
+  )
+
 component :: Component ListProps
 component = createComponent "List"
 
-list :: ListProps -> JSX
-list = makeStateless component $ lumiList <<< mapProps
+listComponent :: ListProps -> JSX
+listComponent = makeStateless component $ lumiList <<< mapProps
```

Comment:
I believe it makes more sense to either keep this as it was (that is, just add `borders` as a prop to `list`), or to keep it named `list` and have it take a `ListProps ()` as props.

The idea we discussed was to completely hide the `borders` prop from the user, so that they would see `list` and `borderlessList` as two different components which are, under the hood, implemented as a single one. Something like:
```purescript
list :: ListProps () -> JSX
list props =
  listComponent
    { size: props.size
    , rightAligned: props.rightAligned
    , rows: props.rows
    , borders: true
    }

borderlessList :: ListProps () -> JSX
borderlessList props =
  listComponent
    { size: props.size
    , rightAligned: props.rightAligned
    , rows: props.rows
    , borders: false
    }

-- The component below is not exported from this module:
listComponent :: ListProps (borders :: Boolean) -> JSX
listComponent = ...
```

This way we can avoid a breaking change, and it makes sense to me that a list with borders and a list without borders are handled as different components. But, please, keep in mind that this is only my personal opinion and a suggestion.

Reply:
Ah I see what you mean, thanks for the break down. Yes, I believe this approach makes sense (a list in my mind has borders, vs. an explicitly borderless list).